### PR TITLE
draft: IX Bid Adapter: change outgoing exchange request type from GET to POST

### PR DESCRIPTION
**Pull request by ix-peiyuan**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

(The PR is a replicate of [#8658](https://github.com/prebid/Prebid.js/pull/8658), whose owned is on vacation and I am taking over)

Changes Adapter request to Exchange from GET to POST to have higher request size limit.
No functional changes for users
Tests updated

Contact [peiyuan.sun@indexexchange.com](mailto:peiyuan.sun@indexexchange.com) for more information.

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@ix-peiyuan looks like there is a linting error here but the code it is referring to looks to be removed?

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const BANNER_TIME_TO_LIVE = 300;
 const VIDEO_TIME_TO_LIVE = 3600; // 1hr
 const NATIVE_TIME_TO_LIVE = 3600; // Since native can have video, use ttl same as video
 const NET_REVENUE = true;
-const MAX_REQUEST_SIZE = 8000;
+const MAX_REQUEST_SIZE = 32000; // IX POST limit
```
Comment (muuki88) [inline modules/ixBidAdapter.js:42]: So you keep the splitting logic in the adapter?

Comment (ix-peiyuan) [reply modules/ixBidAdapter.js:42]: who does splitting mean here? you mean different size limits co exist due to different version of PBJS adopted by pubs?

**Comment by ix-peiyuan** [comment]
> @ix-peiyuan looks like there is a linting error here but the code it is referring to looks to be removed?

I did the fix for the linting error but it seems CI is failing for some test cases belong to `SonobiBidAdapter`?

File: modules/ixBidAdapter.js (hunk 1 above)
Comment (ix-peiyuan) [reply modules/ixBidAdapter.js:42]: Yes, I think it is a good point. I will talk with my team on removing the redundant logic but for now we could keep this as another follow-up change after POST is successfully in live.

**Pull request closed without merging**